src/classes/Standardizer/plugins/Discord/getters/getConnections.ts

```diff
@@ -24,7 +24,7 @@ Discord.prototype.getConnections = withAutoParser(async parser => {
           },
         },
         browser: row.browser,
-        timestamp: new Date(row.timestamp),
+        timestamp: new Date(row.timestamp.slice(1, -1)),
```

Comment:
Use standard Date.parse() method : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

```suggestion
        timestamp: new Date(Date.parse(row.timestamp)),
```